routes/notes.route.js

```diff
@@ -0,0 +1,106 @@
+import express from "express";
+import authMiddleware from '.././middlewares/auth.middleware.js';
+import notesService from '.././services/note.service.js';
+import { body,check,oneOf,param,query, validationResult } from 'express-validator';
+
+const notesRouter = express.Router();
+
+notesRouter.get('/api/notes',
+    async (req,res,next)=>{
+        //request non autenticata
+        //Se la query string non contiene filtri 
+        if(!req.query.date && !req.query.limit)
+            res.status(200)
```

Comment:
usa sempre le {} anche per una sola istruzione.

Reply:
Risolto in nuovo commit